ledger/object/lock.go

```diff
@@ -53,7 +53,7 @@ func NewIndexLocker() IndexLocker {
 // If mutex does not exist, it will be created in concurrent safe fashion.
 func (l *idLocker) Lock(id insolar.ID) {
 	// Reset pulse. It should not be considered when locking.
-	normalizedID := *insolar.NewID(0, id.Hash())
```

Comment:
Leave it at zero. It meant to be a zero-value.

Reply:
We can't have an ID with Zero, now. It's invalid value (UnknownPulse)

Reply:
If you need to store some invalid ID with magic PulseNumber - you should do it some other way.

Reply:
We can add a reserved “nonce” pulse number.

Reply:
discussed, using `pulse.LocalRelative` constant

Reply:
fixed